services/web/client/source/class/osparc/data/Permissions.js

```diff
@@ -138,14 +138,17 @@ qx.Class.define("osparc.data.Permissions", {
           "preferences.user.update",
           "preferences.token.create",
           "preferences.token.delete",
+          "preferences.tag",
```

Comment:
The convention we are following for the permissions is:
scope.resource.action

Reply:
does it make sense to leave it to block all actions inside the preferences view? i am not showing the tab at all...

Reply:
@ignapas Please sync from time to time the front-end permissions with [those defined in the backend](https://github.com/ITISFoundation/osparc-simcore/blob/857de39ba6438e97eda2d913e15aedd470598772/services/web/server/src/simcore_service_webserver/security_roles.py)